base_station/gui/src/components/LCMEcho.vue

```diff
@@ -94,10 +96,12 @@
           '/target_list': false,
           '/temperature': false,
           '/tennis_ball': false,
+          '/thermistor_data': false,
           '/zed_gimbal_cmd': false,
           '/zed_gimbal_data': false
         },
         subscriptions: [
+          {'topic': '/ammonia_cmd', 'type': 'AmmoniaCmd'},
```

Comment:
has it always been topic? did it used to be type?

Reply:
Yes, it's always been topic